x-pack/legacy/plugins/transform/public/app/sections/transform_management/components/transform_list/transform_list.tsx

```diff
@@ -372,25 +376,34 @@ export const TransformList: FC<Props> = ({
 
   return (
     <Fragment>
```

Comment:
Since there's now a wrapping `div` element, you can remove the `Fragment` tag.

Reply:
:+1: Removed in 9e4e6c8